controller/TestPreviewer.php

```
@@ -32,13 +35,15 @@
 use oat\taoQtiTestPreviewer\models\test\TestPreviewRequest;
 use oat\taoQtiTestPreviewer\models\TestCategoryPresetMap;
 use oat\taoQtiTestPreviewer\models\testConfiguration\service\TestPreviewerConfigurationService;
+use oat\taoTests\models\event\TestContentViewEvent;
 use qtism\data\storage\xml\XmlStorageException;
 use tao_actions_ServiceModule;
 use Throwable;
 
 class TestPreviewer extends tao_actions_ServiceModule
```

PHPStan (2.2.7)

[error] Class oat\taoQtiTestPreviewer\controller\TestPreviewer extends unknown class tao_actions_ServiceModule.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)